go-controller/pkg/ovn/policy.go

```diff
@@ -1769,3 +1563,12 @@ func PortGroupHasPorts(nbClient libovsdbclient.Client, pgName string, portUUIDs
 
 	return sets.NewString(pg.Ports...).HasAll(portUUIDs...)
 }
+
+func getStaleNetpolAddrSetDbIDs(policyNamespace, policyName, policyType, idx, controller string) *libovsdbops.DbObjectIDs {
```

Comment:
can we get some comments on this function so in the future we have a reference to what "stale" means here?